shell/platform/android/io/flutter/embedding/engine/systemchannels/TextInputChannel.java

```diff
@@ -110,6 +110,17 @@ public TextInputChannel(@NonNull DartExecutor dartExecutor) {
     channel.setMethodCallHandler(parsingMethodHandler);
   }
 
+  /**
+   * Instructs Flutter to reattach the last active text input client, if any.
+   *
+   * This is necessary when the view heirarchy has been detached and reattached
+   * to a {@link FlutterEngine}, as the engine may have kept alive a text
+   * editing client on the Dart side.
+   */
+  public void reattach() {
```

Comment:
Would it make sense to call this something like `requestExistingInputState()` instead, so that we remove the implicit temporal relationship and clarify why we call it?

Reply:
SGTM.